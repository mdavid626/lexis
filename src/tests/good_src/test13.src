/' Program 13: full function '/

function sm (s as string, i as integer, d as double) as integer
	print s; i; d; "\n";
	return (0 - 1)
end function

scope 
print sm("Ahoj", 0, 2.0); "\n";
end scope
