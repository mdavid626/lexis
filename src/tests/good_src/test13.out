Ahoj 0 2
-1
